src/ResourceManager/Relay/Commands.Relay/Cmdlets/HybridConnections/SetAzureRelayHybridConnections.cs

```diff
@@ -0,0 +1,81 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Relay.Models;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Relay.Commands.HybridConnections
+{
+    /// <summary>
+    /// 'Set-AzureRmRelayHybridConnection' Cmdlet updates the specified WcfRelay
+    /// </summary>
+    [Cmdlet(VerbsCommon.Set, RelayHybridConnectionVerb, SupportsShouldProcess = true), OutputType(typeof(HybridConnectionAttibutes))]
+    public class SetAzureRelayHybridConnections : AzureRelayCmdletBase
+    {
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 0,
+            HelpMessage = "Resource Group Name.")]
+        [ValidateNotNullOrEmpty]
+         public string ResourceGroup { get; set; }
+
+        [Parameter(Mandatory = true,
+           ValueFromPipelineByPropertyName = true,
+           Position = 1,
+           HelpMessage = "Namespace Name.")]
+        [ValidateNotNullOrEmpty]
+        public string Namespace { get; set; }
+
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 2,
+            HelpMessage = "HybridConnections Name.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(Mandatory = false,
+           ParameterSetName = HybridConnectionInputObjectParameterSet,
+           ValueFromPipelineByPropertyName = true,
```

Comment:
If this is an InputObject, it should take ValueFromPipeline, not ValueFromPipelineByPropertyName